puppet/zulip/files/logrotate/zulip

```diff
@@ -1,8 +1,13 @@
-/var/log/zulip/server.log /var/log/zulip/workers.log /var/log/zulip/manage.log {
+/var/log/zulip/*.log {
```

Comment:
Correct, the idea there was to include all file scattered around there. When I took a look, a few week after production usage of Zulip, I don't think I saw those logs being compressed... so I'd still include them, but either way is fine really.
I do still think that even if you managed to reduce the size, those files still need to be kept under monitoring.